zebra-network/src/protocol/external/codec.rs

```diff
@@ -113,6 +123,10 @@ impl Encoder for Codec {
             return Err(Parse("body length exceeded maximum size"));
         }
 
+        if let Some(addr) = self.builder.addr.clone() {
+            metrics::counter!("bytes.written", (body.len() + HEADER_LEN) as u64, "addr" =>  addr);
```

Comment:
```suggestion
            metrics::counter!("bytes.written", (body.len() + HEADER_LEN) as u64, "label" => label);
```
Let's change `addr` to `label` to be consistent with the field name.

Reply:
https://github.com/ZcashFoundation/zebra/pull/901/commits/930f99a7cf190649465cd90631f1326d5507186b